package.json

```
@@ -64,7 +64,7 @@
     "mock-require": "^3.0.3",
     "nyc": "^15.1.0",
     "ts-node": "^10.9.2",
-    "typescript": "^5.3.3"
+    "typescript": "^5.9.3"
   },
   "optionalDependencies": {
     "node-purple": "git+https://github.com/matrix-org/node-purple#1adfe21219863824a1fcb4c1de35b5b44cccca37"
```

Not sure how important this is, but we could also update `node-purple`'s Node support as well.  See https://github.com/matrix-org/node-purple/pull/11
```suggestion
    "node-purple": "git+https://github.com/matrix-org/node-purple#6c711f276772f78edf9d0dc5e1ad442e2291ec81"
```

---

Sure, though generally I would not want to bring in unrelated changes for a bugfix. It will be annoying to debug something breaking.